docs/configuration/logs.md

```diff
@@ -122,20 +124,27 @@ filePath = "/path/to/access.log"
 format = "json"
 ```
 
-To filter logs by status code:
+To filter logs:
 ```toml
 [accessLog]
 filePath = "/path/to/access.log"
 format = "json"
 
   [accessLog.filters]
 
-  # statusCodes keep only access logs with status codes in the specified range
+  # statusCodes keep access logs with status codes in the specified range
   #
   # Optional
   # Default: []
   #
   statusCodes = ["200", "300-302"]
+
+  # retryAttempts keep access logs when at least one retry happened
```

Comment:
Can you add a specify that this option will keep `keep access logs when at least one retry happened` **even if the status code is not in the `statusCodes` ranges**.
Thus users can understand that filters are _OR-connected_ :wink:

Reply:
@nmengin I added now a sentence at the beginning of the segment, so that we don't have to repeat that sentence on every filter we have :D 

Can you have a look and let me know whether it would have helped you?